src/main/java/com/google/api/codegen/config/PackageMetadataConfig.java

```diff
@@ -223,6 +230,7 @@ public static PackageMetadataConfig createFromString(String yamlContents) {
         .packageName(buildMapWithDefault((Map<String, String>) configMap.get("package_name")))
         .shortName((String) configMap.get("short_name"))
         .packageType(PackageType.of((String) configMap.get("package_type")))
+        .generationLayer(GenerationLayer.of((String) configMap.get("generation_layer")))
```

Comment:
Do you think it makes sense to combine the generation_layer with package_type? (e.g. we can have something like raw_proto, grpc etc)